ccsrc/lib/cppsim/CMakeLists.txt

```diff
@@ -0,0 +1,305 @@
+cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
+
+project(
+  cppsim
+  VERSION 1.0.0
+  LANGUAGES C CXX)
+
+# Minimum required Python version (used both in this file and in the installed CMake configuration)
+set(CPPSIM_PYTHON_VERSION_MIN 3.7.0)
+
+option(BUILD_TESTING "Build the test suite?" OFF)
+
+if(NOT DEFINED ITERATOR_DEBUG_VALUE)
+  set(ITERATOR_DEBUG_VALUE 0)
+endif()
+option(MSVC_ITERATOR_DEBUG "Define _ITERATOR_DEBUG_LEVEL (defaults to 0, can be set using ITERATOR_DEBUG_VALUE)" OFF)
```

Comment:
To match with the rest of MindQuantum, remove these lines. The CMake options and variables should be already defined by MindQuantum